### ISSUE TITLE
about testloader

### ISSUE THREAD

**Issue by jiangyanjavawin** [xufangchn/SBRSIR#1]
hello sir,Thanks for your amazing code,It's very good,when i watch the code,i have some question about the meaning of test_dataset = {x: datasets.ImageFolder(os.path.join(data_dir, 'test', x), data_transforms['test']) for x in ['query', 'database']} and 
test_loader = {x: torch.utils.data.DataLoader(test_dataset[x], batch_size=batch_size, shuffle=False, num_workers=1) for x in ['query', 'database'],what's the meaning of query and database?Is the thing of list is the class?thank you very much.

**Comment by xufangchn** [comment]
Hi, the query means the sketches, and the database means the remote sensing images that to be retrieved.

**Comment by xufangchn** [comment]
> hello sir,Thanks for your amazing code,It's very good,when i watch the code,i have some question about the meaning of test_dataset = {x: datasets.ImageFolder(os.path.join(data_dir, 'test', x), data_transforms['test']) for x in ['query', 'database']} and
> test_loader = {x: torch.utils.data.DataLoader(test_dataset[x], batch_size=batch_size, shuffle=False, num_workers=1) for x in ['query', 'database'],what's the meaning of query and database?Is the thing of list is the class?thank you very much.

Hi, the query means the sketches, and the database means the remote sensing images that to be retrieved.

**Comment by jiangyanjavawin** [comment]
> > hello sir,Thanks for your amazing code,It's very good,when i watch the code,i have some question about the meaning of test_dataset = {x: datasets.ImageFolder(os.path.join(data_dir, 'test', x), data_transforms['test']) for x in ['query', 'database']} and
> > test_loader = {x: torch.utils.data.DataLoader(test_dataset[x], batch_size=batch_size, shuffle=False, num_workers=1) for x in ['query', 'database'],what's the meaning of query and database?Is the thing of list is the class?thank you very much.
> 
> Hi, the query means the sketches, and the database means the remote sensing images that to be retrieved.

thank you very much!

**Comment by jiangyanjavawin** [comment]
> > hello sir,Thanks for your amazing code,It's very good,when i watch the code,i have some question about the meaning of test_dataset = {x: datasets.ImageFolder(os.path.join(data_dir, 'test', x), data_transforms['test']) for x in ['query', 'database']} and
> > test_loader = {x: torch.utils.data.DataLoader(test_dataset[x], batch_size=batch_size, shuffle=False, num_workers=1) for x in ['query', 'database'],what's the meaning of query and database?Is the thing of list is the class?thank you very much.
> 
> Hi, the query means the sketches, and the database means the remote sensing images that to be retrieved.

thank you very much!
